BorrowBookControl.java

```diff
@@ -97,7 +97,7 @@ public void commitLoans() {
         if (!this.state.equals(ControlState.FINALISING)) {
             throw new RuntimeException("BorrowBookControl: cannot call commitLoans except in FINALISING state");
         }
-        for (book borrowBookList : this.borrowBookStatus) {
+        for (book borrowBookList : this.booksPendingBorrow) {
             loan LOAN = this.library.ISSUE_LAON(borrowBookList, this.member);

```

Comment:
D15: refactor variable name 'LOAN' to meet code style guidelines.

Reply:
done